### PR TITLE
Fixes space lattice building

### LINKED ISSUE #24673: You cannot place a lattice on a space tile
Confirmed by ausops and I when I accidentally deconned one of the solar catwalks during a carp fight. Does not work as both Human and cyborg. You can place catwalks on existing lattices but you cannot make new ones.

### PR DESCRIPTION
**Pull request by Cyberboss**
Fixes #24673

:cl: Cyberboss
fix: You can build lattice in space again
/:cl:

### CODE REVIEW

**Comment by KorPhaeron** [comment]
here we go apc destroyed mission accomplished

**Comment by factoryman942** [comment]
yay

File: code/game/turfs/space/space.dm [hunk 1]
```diff
@@ -60,10 +60,10 @@
 /turf/open/space/attack_paw(mob/user)
 	return src.attack_hand(user)
 
-/turf/open/space/attackby(obj/item/C, mob/user, area_restriction)
+/turf/open/space/attackby(obj/item/C, mob/user, area/area_restriction)
 	..()
 	if(istype(C, /obj/item/stack/rods))
-		if(area_restriction && !istype(get_area(src), area_restriction))
+		if(istype(area_restriction) && !istype(get_area(src), area_restriction))
```
Comment (RemieRichards) [inline code/game/turfs/space/space.dm]: Shit fix. The third argument to ```attackby()``` is the Click params, this is why things didn't work, move your crap to be the 4th arg.

**Comment by fizbump** [comment]
As of right now on Sybil, lattice building still doesn't work.

**Comment by Cyberboss** [comment]
Check the revision data commit

**Pull request merged**